SettingsView/AiForms.SettingsView.csproj

```diff
@@ -0,0 +1,52 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.22">
+
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;xamarin.ios10;monoandroid10.0</TargetFrameworks>
+        <LangVersion>latest</LangVersion>
+        <Nullable>enable</Nullable>
+        <DisableExtraReferences>False</DisableExtraReferences>
+        <PackageProjectUrl>https://github.com/muak/AiForms.SettingsView</PackageProjectUrl>
+        <Product>$(AssemblyName) ($(TargetFramework))</Product>
+        <Authors>kamu</Authors>
+        <Description>This is a flexible TableView specialized in settings for Android / iOS.
+    There are various cells such as (LabelCell,ButtonCell,CommandCell,SwitchCell,CheckboxCell,RadioCell,PickerCell,EntryCell,NumberPickerCell,TimePickerCell,DatePickerCell,CustomCell)
+        </Description>
+        <PackageTags>Xamarin.Forms TableView Cell Setting Configuration Option ListView UITableView RecyclerView ReOrder DragDrop</PackageTags>
+        <VersionPrefix>1.5.0</VersionPrefix>
+        <VersionPrefix Condition=" $(BUILD_BUILDNUMBER) != '' ">$(VersionPrefix).$(BUILD_BUILDNUMBER)</VersionPrefix>
+        <PackageReleaseNotes></PackageReleaseNotes>
+        <PackageIconUrl>https://raw.githubusercontent.com/muak/AiForms.SettingsView/master/images/icon.png</PackageIconUrl>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <LangVersion>latest</LangVersion>
+        <RepositoryUrl>https://github.com/muak/AiForms.SettingsView</RepositoryUrl>
+        <RepositoryType>git</RepositoryType>
+        <DefineConstants Condition=" $(TargetFramework.StartsWith('monoandroid10'))">$(DefineConstants);ANDROIDX</DefineConstants>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" $(Configuration) == 'Debug' ">
+        <DebugType>full</DebugType>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" $(Configuration) == 'Release' ">
+        <DebugType>embedded</DebugType>
+        <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <Compile Remove="Platforms\**\*.*" />
+        <None Include="Platforms\**\*.*" />
+        <PackageReference Include="Xamarin.Forms" Version="5.0.0.2012" />

```

Comment:
will this cause a conflict with those folks that depend on AIForms.SettingsView but cannot, for various reasons, move to Xamarin.Forms 5?

I wonder if this PR is making too many assumptions.

Reply:
The PR isn't making any assumptions.  It's what works for me.